web_console/tests/libs/es_test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const ElasticSearchClient = require('../../libs/es');
+
+let es_client;
+
+describe('ElasticSearch Client', () => {
+    before(() => {
```

Comment:
`before` hook is used for heavy setup, e.g: start a k8s cluster, es client could be simply assigned as a constant like above